source/how-tos/create-ias-boot-image.rst

```diff
@@ -0,0 +1,116 @@
+.. _create-ias-boot-image:
+
+Create IAS Boot Image

```

Comment:
Personally, I think the tense of the title should be continuous, e.g. "Creat**ing** an IAS Image"

Reply:
I chose to make all "how-to" titles to use present tense, as they are shorter on the left column for navigation.